test/archive.cpp

```diff
@@ -518,6 +518,11 @@ TEST(ZimArchive, validate)
     "Invalid cluster pointer\n"
   );
 
+  TEST_BROKEN_ZIM_NAME(
```

Comment:
Commenting on the commit description:

> Nons variant detect the invalid offset as `urlPtrList` is stored in the cluster and so we try to open it.

`urlPtrList` cannot be stored in a cluster.

Reply:
Indeed. Commit message directly fixed.